src/custom/ecospheres/views/HomeView.vue

```diff
@@ -185,13 +185,13 @@ useHead({
     </section>
     <section class="fr-container fr-py-12v">
       <h2>Les bouquets à découvrir</h2>
-      <ul class="fr-grid-row fr-grid-row--gutters" role="list">
+      <ul class="fr-grid-row fr-grid-row--gutters fr-mb-2w" role="list">
         <li
           v-for="topic in lastTopics"
           :key="topic.id"
           class="fr-col-12 fr-col-xl-4"
         >
-          <BouquetCardHome :bouquet="topic" />
+          <BouquetCard :bouquet="topic" hide-description="True" />
```

Comment:
Utiliser un booléen ici non ?

```suggestion
          <BouquetCard :bouquet="topic" :hide-description=true" />
```